react/features/video-status-label/components/VideoStatusLabel.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import AudioOnlyLabel from './AudioOnlyLabel';
+import HDVideoLabel from './HDVideoLabel';
+
+/**
+ * React {@code Component} responsible for displaying a label that indicates
+ * the displayed video state of the current conference. {@code AudioOnlyLabel}
+ * will display when the conference is in audio only mode. {@code HDVideoLabel}
+ * will display if not in audio only mode and a high-definition large video is
+ * being displayed.
+ */
+export class VideoStatusLabel extends Component {
+    /**
+     * {@code VideoStatusLabel}'s property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * Whether or not the conference is in audio only mode.
+         */
+        _audioOnly: React.PropTypes.bool,
+
+        /**
+         * Whether or not a high-definition large video is displayed.
+         */
+        _largeVideoHD: React.PropTypes.bool
+    }
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement|null}
+     */
+    render() {
+        if (this.props._audioOnly) {
+            return <AudioOnlyLabel />;
+        }
+
+        if (this.props._largeVideoHD) {
```

Comment:
I don't think it's possible because of the early returns. I can change the syntax to an else/if though to make it clear it's an either/or situation and a default of null outside the two cases.